### ISSUE TITLE
Feature Request: Sell to Merchant

### ISSUE THREAD

**Issue by vvZODvv** [foundryvtt/pf2e#7598]
The merchant sheet is awesome for purchasing - the player drags the item onto their sheet and the appropriate currency is deducted!  

When selling, currently players can drag items off their character sheet and it will populate on the merchant sheet, but no currency on the player sheet is added.  Please consider further automating the buying/selling process by having half the value of the item (per CRB 502: Buying and Selling) populate onto the player sheet during the transfer process.

**Comment by TMunsie** [comment]
The problem is that the CRB says it *typically* sells for that price. It doesn’t *always* sell for that price. It is up to the GM.

**Comment by CarlosFdez** [comment]
Needs a bit more features for it to work well like an items on offer section that the GM signs off on I think.

**Comment by vvZODvv** [comment]
Good point TMun.  Keeping in mind I am only spit balling ideas and have no background in coding, therefore don't know the difficulty of the request, but perhaps a slider or text field box the GM types in the %.

The Sell box defaults to 50%, but can be adjusted.
The Buy box defaults to 100%, but likewise be adjusted (such as in Age of Ashes, when items sell for 5% cheaper in Breachill).

**Comment by PropaneMilo** [comment]
Would it be possible (and/or practical) for players to push items into a vendor that triggers a popup for the Gamemaster?

The popup would include a list of all items being sold along with a slider ranging from 0% value to 100%, with the default for particular items being adjustable.

**Comment by pontusedberg** [comment]
I would very much love the option for players to Sell other junk besides Treasure Directly to merchant sheet.
(they love picking up those Daggers from goblins, and try to sell em) 
If you can set up the Merchant sheet in the same way that Monks enhanced journal does it, That would be amazing
(where you can set how much a merchant are willing to pay, and also pay for a specific item)
We could also automate the sheet by checking how much an impression the player have on that merchant, to be able to sell to em for a higher price?

**Comment by pontusedberg** [comment]
![Uploading MerchantSheet1.png…]()

**Comment by pontusedberg** [comment]
> Until there are formal rules from Paizo on the mechanics this is likely a better fit for a module. Though if you were to code it we would certainly evaluate it for inclusion. That’s not a guarantee though

Having the ability to at least Sell items to merchants would be great <3
https://2e.aonprd.com/Rules.aspx?ID=937

**Comment by TMunsie** [comment]
> > Until there are formal rules from Paizo on the mechanics this is likely a better fit for a module. Though if you were to code it we would certainly evaluate it for inclusion. That’s not a guarantee though
> 
> Having the ability to at least Sell items to merchants would be great <3 https://2e.aonprd.com/Rules.aspx?ID=937

Please read my first comment: there aren’t codified rules for this in the CRB or the GMG. The link you posted effectively says “run merchants however you want. If you don’t want that level of difficulty typically it is half price. That’s not firm enough rules to codify.

You are welcome to submit code on the topic that would add similar function to the system for review.
